### DISCUSSION TITLE
使用的群晖NAS DS416j不支持docker，从哪儿能直接获取DSM的alist套件？

### DISCUSSION THREAD

**Discussion by l21565** [AlistGo/alist#2251] [Q&A]
RT，找了几天了，也没找到。

**Comment by l21565** [comment]
忘记说了，DSM是7.0版本

**Comment by zhouyb73** [comment]
矿神群晖spk套件中心

**Comment by l21565** [comment]
多谢答复，不过我的DS416J好像不支持这个套件，社群中看不到。后来我在linux下直接install了armv7发布版本，成功了。但另一台DS212J安装arm5发布版本，运行失败，可能是编译器不支持。



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年11月08日 09:49 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [alist-org/alist] 使用的群晖NAS DS416j不支持docker，从哪儿能直接获取DSM的alist套件？ (Discussion #2251) |

矿神群晖spk套件中心
